### DISCUSSION TITLE
SPM - How can I add more products after initial add package?

### DISCUSSION THREAD

**Discussion by viction** [firebase/firebase-ios-sdk#9105] [Q&A] [answered]
This feels like a noob question... but I'm totally stumped.  I've migrated my project from Pods to SPM - successfully installed Firebase products with SPM with no issues, however, I now want to add FirebaseFunctions (which I didn't initially install) -- and there seems to be no option to install additional products.  I get a message that my project already contains this (firebase-ios-sdk) package dependency. Do I have to uninstall and reinstall firebase-ios-sdk and am limited to what I choose at time of "Add Package?" to get more products?  Or is there a config file I'm missing (like cocoapods had Podfile).  Thanks for any help and sorry if this is a basic question, but I really can't find information on it.  I recently upgraded to xCode 13.2 - so any info out there seems dated.

**Comment by viction** [comment, marked as answer]
I found where to add additional products in this article [Adding and Updating Swift Package Products with Xcode](https://www.kiloloco.com/articles/010-add-package-product-xcode/)

**Reply by StarboySupernova** [reply]
Thanks, really helpful

**Reply by ozgurshn** [reply]
Thank you

**Reply by peterfriese** [reply]
This is also covered in our [intro video](https://youtu.be/F9Gs_pfT3hs?t=738) at the 12:18 mark.

**Reply by cabyambo** [reply]
This was amazing! Wish it was easier to find in the docs

**Reply by vsiisv** [reply]
Thanks a lot

**Reply by Ameya-Joshi79** [reply]
Oh wow! I spent so much time with this. Was about to install it as a pod. Thank you so much! This was so easy!

**Reply by Matheus-C-S-Azevedo** [reply]
Thank you so much 🇧🇷
